test/integration/targets/openvswitch_bridge/tests/basic.yaml

```
@@ -22,5 +22,27 @@
     that:
       - "result.changed == false"
 
-- name: Tear down test bridge
+- name: Add fake bridge
+  openvswitch_bridge:
+    bridge: fake-br-test
+    parent: br-test
+    vlan: 100
+  register: result
+
+- assert:
+    that:
+      - "result.changed == true"
```

no need for quotes in this case..
```suggestion
      - result.changed == true
```

---

Oh, wait, also no need for the comparison with true: first of all, the operator should be `is` and second -- it's already truthy.
```suggestion
      - result.changed
```